wpilibc/src/main/native/cpp/Compressor.cpp

```diff
@@ -82,4 +84,14 @@ void Compressor::InitSendable(wpi::SendableBuilder& builder) {
       "Enabled", [this] { return IsEnabled(); }, nullptr);
   builder.AddBooleanProperty(
       "Pressure switch", [this] { return GetPressureSwitchValue(); }, nullptr);
+  builder.AddDoubleProperty(
+      "Current (A)", [this] { return GetCurrent().value(); }, nullptr);
+  // These are not supported by the CTRE PCM
+  if (dynamic_cast<PneumaticHub*>(m_module.get())) {
```

Comment:
Check `getModuleType` rather than casting

Reply:
where do you see this method?

Reply:
It's `getType`, not `getModuleType`.

Reply:
I still can't seem to find this method

Reply:
My bad, I was looking at the PowerDistribution class. It looks like you're right in that there's no way to get the module type except for casting or storing the module type in a variable. In that case, you can just store the module type as a variable.